recipes/service.rb

```diff
@@ -21,7 +21,12 @@
 # systemd platforms use an instance service
 case node['platform_family']
 when 'rhel'
-  if node['platform_version'] >= '7'
+  if node['platform_version'] >= '8'
```

Comment:
```suggestion
  if node['platform_version'].to_i >= 8
```

For compatibility with Chef < 16 you'll want to make sure you compare this as an integer